var/spack/repos/builtin/packages/ncbi-rmblastn/package.py

```diff
@@ -26,11 +26,102 @@
+
+    patch('isb-2.6.0+-changes-vers2.patch', when='@2.6.0', level=1)
 
     configure_directory = 'c++'
+    build_directory = 'c++'
+
+    def configure_args(self):
+        if self.spec.version < Version('2.6.0'):
+            options = []
+        elif self.spec.version == Version('2.6.0'):
+            options = [
+                '--without-internal',
+                '--with-mt',
+                '--without-debug',
+                '--without-krb5',
+                '--without-check',
+                '--without-bz2',
+                '--without-lzo',
+                '--with-strip',
+                '--with-ncbi-public',
+                '--without-ncbi-c',
+                '--without-sss',
+                '--without-sssdb',
+                '--without-vdb',
+                '--without-pcre',
+                '--without-gcrypt',
+                '--without-nettle',
+                '--without-gnutls',
+                '--without-openssl',
+                '--without-sybase',
+                '--without-ftds',
+                '--without-mysql',
+                '--without-opengl',
+                '--without-mesa',
+                '--without-glut',
+                '--without-glew',
+                '--without-wxwidgets',
+                '--without-freetype',
+                '--without-ftgl',
+                '--without-fastcgi',
+                '--without-bdb',
+                '--without-sp',
+                '--without-orbacus',
+                '--without-sqlite3',
+                '--without-icu',
+                '--without-expat',
+                '--without-sablot',
+                '--without-libxml',
+                '--without-libxslt',
+                '--without-libexslt',
+                '--without-xerces',
+                '--without-xalan',
+                '--without-zorba',
+                '--without-oechem',
+                '--without-sge',
+                '--without-muparser',
+                '--without-hdf5',
+                '--without-gif',
+                '--without-png',
+                '--without-tiff',
+                '--without-xpm',
+                '--without-magic',
+                '--without-curl',
+                '--without-mimetic',
+                '--without-gsoap',
+                '--without-avro',
+                '--without-cereal',
+                '--without-sasl2',
+                '--without-mongodb',
+                '--without-gmock',
+                '--without-lapack',
+                '--without-lmdb',
+                '--without-3psw',
+                '--without-local-lbsm',
+                '--without-ncbi-crypt',
+                '--without-connext',
+                '--without-dbapi',
+                '--without-ctools',
+                '--without-gui',
+                '--without-gbench'
+            ]
+        return options
+
+    def install(self, spec, prefix):
```

Comment:
you can override this for only some versions by using the `@when` decorator. For usage see e.g. the `nest` package